docs/en/oas3/OAS3-PDND-AS.yaml

```
@@ -0,0 +1,313 @@
+          required: true
+          schema:
+            type: string
+            example: SHA-256=b626aa33e25d46086dd5c12b631be51caeece92689912408e96dc4a430f3fa67
+        - name: Agid-JWT-TrackingEvidence
+          in: header
+          description: >-
+            JWT containing the data tracker in the Consumer's domain. It is mandatory only when complying with AUDIT_REST_02.
+            <a target="blank"
+            href="https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/e-service-pdnd.html">e-Service PDND</a>. <br/><br/>
+
+            <a target="blank" href="https://jwt.io/#debugger-io?token=eyJhbGciOiJFUzI1NiIsImtpZCI6IjhlOGRlNGJmZTFlNWVhNzkwM2E1ODE2ZmNlNzNjZDQ4IiwidHlwIjoiSldUIn0.eyJpc3MiOiJpbnRlcm9wLnBhZ29wYS5pdCIsImF1ZCI6Imh0dHBzOi8vZXJvZ2F0b3JlLmV4YW1wbGUvZW50ZS1leGFtcGxlL3YxIiwiZXhwIjoxNzgzMDQyMTUwLCJuYmYiOjE3MzMwNDE5NDUsImlhdCI6MTczMzA0MTkyMCwianRpIjoiYzRmNWQ3ZTItYjdjOC00MGY2LTliNmEtZGM5YTRmNWFlYjU3IiwiY2xpZW50X2lkIjoiODI5MTRiM2YtNjBiMi00NTI5LWI0ZDYtM2Q0ZTY3ZjBhOTMzIiwicHVycG9zZUlkIjoiZDJiOWE2NTMtYzQ5Ny00NWM2LWI4ZjEtNWJkZjEyNGM5ZDNhIiwiZG5vbmNlIjoiNjUyODQyNDIxMzY4NSIsInVzZXJJRCI6ImE4YjdjNmQ1LWU0ZjMtZzJoMS1pOWowLWtsbW5vcHFyc3R1diJ9.vHispWBZeWk0UYwUSMAxpioC4giifbctyC3sgiqNxNHt_ypxIr6FZkKlTrLc02WivSm3ApbZ0ABi1RgtcUthYQ">EXAMPLE
+            ON JWT.IO</a>
+          required: true
+          schema:
+            type: string
+            format: JWT
+            example: eyJhbGciOiJFUzI1NiIsImtpZCI6IjhlOGRlNGJmZTFlNWVhNzkwM2E1ODE2ZmNlNzNjZDQ4IiwidHlwIjoiSldUIn0.eyJpc3MiOiJpbnRlcm9wLnBhZ29wYS5pdCIsImF1ZCI6Imh0dHBzOi8vZXJvZ2F0b3JlLmV4YW1wbGUvZW50ZS1leGFtcGxlL3YxIiwiZXhwIjoxNzgzMDQyMTUwLCJuYmYiOjE3MzMwNDE5NDUsImlhdCI6MTczMzA0MTkyMCwianRpIjoiYzRmNWQ3ZTItYjdjOC00MGY2LTliNmEtZGM5YTRmNWFlYjU3IiwiY2xpZW50X2lkIjoiODI5MTRiM2YtNjBiMi00NTI5LWI0ZDYtM2Q0ZTY3ZjBhOTMzIiwicHVycG9zZUlkIjoiZDJiOWE2NTMtYzQ5Ny00NWM2LWI4ZjEtNWJkZjEyNGM5ZDNhIiwiZG5vbmNlIjoiNjUyODQyNDIxMzY4NSIsInVzZXJJRCI6ImE4YjdjNmQ1LWU0ZjMtZzJoMS1pOWowLWtsbW5vcHFyc3R1diJ9.vHispWBZeWk0UYwUSMAxpioC4giifbctyC3sgiqNxNHt_ypxIr6FZkKlTrLc02WivSm3ApbZ0ABi1RgtcUthYQ
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CredentialClaimsRequest'
+        required: true
+      responses:
+        '200':
+          description: 200 OK
+          content:
+            application/jwt:
+              schema:
+                $ref: '#/components/schemas/CredentialClaimsResponse'
+              example: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6ImE4ZDEzYzQ1ZWYyZTI4MzBmN2M0NGJkNDk0MWNjYSJ9.eyJpc3MiOiJodHRwczovL3ByZC5hcGkuZ2F0ZXdheS5pcHpzLml0L3QvaXR3YWxsZXQuYXBpL3V0aWxpdHktd2FsbGV0LzEuMC4wL3YxL25vdGlmeVN0YXR1c0NyZWRlbnRpYWxzIiwibmJmIjoxNzM2ODQ2Njg4LCJleHAiOjE3MzY4NDY5MjgsImlhdCI6MTczNjg0NjY4OCwiYXVkIjoiMzE2NzAwOTItZWVjMC00Zjk1LTg4ZGEtZTFjN2NlNWU0NTA1IiwianRpIjoiYzhiZDhhMmYtZTk5MC00NGZhLTkwMTMtMWIzNTNiZmM1YTBkIn0.ExthZCx8LHMlTTtBm9qA8Y3FhlKvTl-_sWXq35e_vTRMmKukrLhBhNa-C1xfli8g_JBixYhsBzUWhE0ING_kIQ'
```

The token detailed in the example is not correct:
- the `iss` claim must represent the specific e-Service (`https://authentic-source.example.it`);
- the `aud` claim must represent the specific Consumer who made the original request (thus must have the same value as the `iss` of the `Signature` and `TrackingEvidence` JWTs).

Moreover, the `kid` is not in the UUID format, thus generating inconsistencies with the other examples.

The correct token is suggested below.

```suggestion
              example: 'eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiIsImtpZCI6IjRlNTAzYjU0LWNiZDUtNDZkOC1iNzhhLTAxMTY5OTEyMmYzMCJ9.eyJpc3MiOiJodHRwczovL2F1dGhlbnRpYy1zb3VyY2UuZXhhbXBsZS5pdCIsIm5iZiI6MTczNjg0NjY4OCwiZXhwIjoxNzM2ODQ2OTI4LCJpYXQiOjE3MzY4NDY2ODgsImF1ZCI6IjgyOTE0YjNmLTYwYjItNDUyOS1iNGQ2LTNkNGU2N2YwYTkzMyIsImp0aSI6ImM4YmQ4YTJmLWU5OTAtNDRmYS05MDEzLTFiMzUzYmZjNWEwZCJ9.4BgoaKyVOupA67tXLQeIK8QNEiYkB646_35HndTkWxS9xypF7FJqyqV24z6EJirSgn5BlT2ZrgqeDURSjJuPUg'
```